pkgs/applications/version-management/mercurial/default.nix

```diff
@@ -50,7 +50,8 @@ in python2Packages.buildPythonApplication {
       chmod u+x $out/share/cgi-bin/hgweb.cgi
 
       # install bash completion
```

Comment:
And refresh that comment to `install shell completions` or `install bash/zsh completions`